web/modules/custom/dpl_fbs/src/FbsApiFactory.php

```diff
@@ -50,4 +51,14 @@ public function getAgencyApi(string $token): ExternalV1AgencyidApi {
     );
   }
 
+  /**
+   * Generate a patron API instance.
+   */
+  public function getPatronApi(string $token): ExternalAgencyidPatronsApi {
```

Comment:
As I wrote in an earlier comment I think it would be nice if the factory gets it token by itself. Could it be an idea to do it in a shared baseClass?